client/src/components/pages/StudentProfile/index.css

```
@@ -0,0 +1,10 @@
+.schoolName {
+    padding: 80px 15px 0 15px;
+  }
+  .schoolName__tiltle {
```

add a line inline 4
and fix indentation from 4 to 9
```suggestion
.schoolName__tiltle {
```

---

ok , I will edit it